src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,18 @@
+package study;
+
+public class StringAddCalculator {
+
+    private static final int NUM_0 = 0;
+    private static String[] numbers;
+    public static int splitAndSum(String text) {
+        if(Validator.isNull(text) || Validator.isEmpty(text)){
+            return NUM_0;
+        }
+
+        numbers= StringUtils.split(text);
+
+        return Calculator.sumNumbers(numbers);
```

Comment:
```suggestion
        return Calculator.sumNumbers(StringUtils.split(text));
```
이렇게 해주면 numbers 필드가 없어도되지 않을까요..?